### PR TITLE
Panos threat (#7746)

### PR DESCRIPTION
**Pull request by bakatzir**
* Added basic requests_mock fixture

* Updated to add command and unit testing

* Further improved yml files and fixed lint issues

* Added basic requests_mock fixture

* Updated to add command and unit testing

* Further improved yml files and fixed lint issues

* Added release notes

* Updated to add outputs and slightly better results calls

* Rectified return val for inner function

* Update Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

* Update Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

* Add release notes and fix whitespace issues

Co-authored-by: Bar Katzir <37335599+bakatzir@users.noreply.github.com>

## Status
- [x] Ready

## Does it break backward compatibility?
   - [x] No

internal contrib

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -4718,6 +4718,73 @@ def panorama_add_static_route_command():
...
+    target = str(demisto.args()['target']) if 'target' in demisto.args() else None
+    result = panorama_get_predefined_threats_list(target)
+    demisto.results(fileResult('predefined-threats.json', json.dumps(result['response']['result']).encode('utf-8')))
+
+
+def panorama_block_vulnerability():
+    """
+    Ovverride a vulnerability signature such that it is in block mode
+    """
+    threatid = demisto.args()['threat_id']
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4769]:
```suggestion
    threatid = demisto.args().get('threat_id')
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -4718,6 +4718,73 @@ def panorama_add_static_route_command():
...
+    vulnerability_profile = demisto.args()['vulnerability_profile']
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4770]:
```suggestion
    vulnerability_profile = demisto.args().get('vulnerability_profile')
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -4718,6 +4718,73 @@ def panorama_add_static_route_command():
...
+    result = http_request(
+        URL,
+        'GET',
+        params=params
+    )
+    return result
+
+
+def panorama_get_predefined_threats_list_command():
+    target = str(demisto.args()['target']) if 'target' in demisto.args() else None
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4760]: why not just use get? dose it make a difference to put here None and not ''?
```suggestion
    target = str(demisto.args().get('target',''))
```

Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:4760]: Yes. it needs to be a None if not existing and this is the convention already in the integration code.

Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:4760]: example:
Screen Shot 2020-07-07 at 9 30 29

**Comment by bakatzir** [comment]
fails on non related `PAN-OS Create Or Edit Rule Test (Mock Disabled)` which fails on nightly.
force merging.

**Pull request merged**